lighthouse-core/test/computed/critical-request-chains-test.js

```diff
@@ -116,7 +116,7 @@ describe('CriticalRequestChain gatherer: extractChain function', () => {
     }
   );
 
-  it('returns correct data for two parallel chains', () => {
```

Comment:
these tests were mostly 3-4 years old when this audit was very different, no real-world trace I found had a situation where this was desirable